doc/getting_going.rst

```diff
@@ -21,48 +21,48 @@ PSyclone directory itself) will be refered to as <PSYCLONEHOME>
 Dependencies
 ------------
 
-PSyclone is written in python so needs python to be installed on the
-target machine. PSyclone has been tested under python 2.6.5 and 2.7.3.
+PSyclone is written in Python so needs Python to be installed on the
+target machine. PSyclone has been tested under Python 2.6.5 and 2.7.3.
 
-PSyclone immediately relies on two external libraries, f2py and
-pyparsing. In addition, f2py requires numpy. To run the test suite you
-will require py.test.
+PSyclone immediately relies on two external Python packages; fparser
+and pyparsing. In addition, fparser requires numpy. To run the test
+suite you will require py.test. The easiest way to satisfy these
+dependencies is to use the Python Package Index (pypi.org) and
+``pip``. See https://packaging.python.org/installing/ for more
+information.
 
 System-specific set-up
 ^^^^^^^^^^^^^^^^^^^^^^
 
 System-specific set-up instructions are available for the following systems
 
-* :ref:`Ubuntu14.03.3`
+* Ubuntu 14.03.3 :ref:`Ubuntu14.03.3`
 
-f2py quick setup
-^^^^^^^^^^^^^^^^
-
-The source code of f2py (revision 93) is provided with PSyclone in the
-sub-directory ``f2py_93``.
+fparser
+^^^^^^^
 
-To use f2py provided with PSyclone you can simply set up your
-PYTHONPATH variable to include this directory.
+The fparser package (https://github.com/stfc/fparser) is a Fortran
+parser originally developed as a part of the f2py project. PSyclone
+requires version >= 0.0.2. It is available from the Python Package
+Index and thus may be installed using pip
+(https://packaging.python.org/installing/#requirements-for-installing-packages):
 ::
 
-    > export PYTHONPATH=<PSYCLONEHOME>/f2py_93:${PYTHONPATH}
+    > pip install --user fparser
 
-If for some reason you need to install f2py yourself then 
-see :ref:`sec_f2py_install`.
+(See :ref:`install_fparser` for more details.)
 
 pyparsing
 ^^^^^^^^^
 
 PSyclone requires pyparsing, a library designed to allow parsers to be be
 built in Python. PSyclone uses pyparsing to parse fortran regular
-expressions as f2py does not fully parse these, (see
+expressions as fparser does not fully parse these, (see
 http://pyparsing.wikispaces.com for more information).
 
-PSyclone has been tested with pyparsing version 1.5.2 which is a
-relatively old version but is currently the version available in the
-Ubuntu software center.
+PSyclone has been tested with pyparsing versions 1.5.2 and 2.0.1.
```

Comment:
After adding this line, a later line no longer makes sense so should be deleted ...
"Versions higher than 1.5.2 should work but have not been tested"